src/registers/model_specific.rs

```diff
@@ -112,6 +121,26 @@ bitflags! {
     }
 }
 
+bitflags! {
+    /// Configurable (RW) flags stored in IA32_U_CET
+    pub struct UCetFlags: u64 {
```

Comment:
The IA32_S_CET MSR uses the same flag format. Perhaps this struct should be renamed to just `CetFlags`.